TomorrowDiesToday.Models/GameModel.cs

```diff
@@ -7,10 +7,10 @@ public class GameModel : IModel
     {
         public string GameId { get; set; }
 
-        public PlayerModel MyPlayer { get; set; }
+        public PlayerModel ThisPlayer { get; set; }
 
-        public List<PlayerModel> OtherPlayers { get; set; }
+        public Dictionary<string, PlayerModel> OtherPlayers { get; set; }
 
-        public List<TileModel> Tiles { get; set; }
+        public Dictionary<string, TileModel> Tiles { get; set; }
```

Comment:
For awareness, LINQ allows you to bypass the complexities and annoyances of dealing with dictionaries.  You could have left this a List<T> and then, when requesting data, used the following code.  I only make note of this because it is so useful and greatly simplifies the interaction and code readability.  You may, of course, leave it as it is.

Tiles.Where(tile => tile.Id == 1).FirstOrDefault();  (will either be the instance or null)
Tiles.Where(tile => tile.IsActive).ToList();